### PR TITLE
Update readme style

### PR DESCRIPTION
**Pull request by davidchesnut**
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | no                                |
| New feature?    | no                                |
| New sample?     | no                                |
| Related issues? | none |

## What's in this Pull Request?

This is to improve the readability of the readme by changing order of sections and removing unnecessary or duplicate information.

### CODE REVIEW

File: Samples/office-contextual-tabs/README.md [hunk 1]
```diff
@@ -17,44 +17,26 @@ description: "Learn how to create a contextual tab that displays on the ribbon i
...
 # Create custom contextual tabs on the ribbon
 
-This sample shows how to create a custom contextual tab on the ribbon in the Office UI. The sample creates a table, and when the user moves the focus inside the table, the custom tab is displayed. When the user moves outside the table, the custom tab is hidden.
+This sample accomplishes the following tasks using Office ribbon APIs.
+
+- Creates a custom contextual tab named **Table Data**.
+- Creates a table in Excel. When the focus is inside the table, the custom tab is displayed.
+- When the focus is outside the table, the custom tab is hidden.
+
+![Screen shot that shows when a table in Excel has the focus, a custom contextual tab named Table Data is shown on the ribbon](pnp-add-contextual-tabs-to-your-add-in.png)
```
Comment (ElizabethSamuel-MSFT) [inline Samples/office-contextual-tabs/README.md:26]:
```suggestion
![Screenshot that shows when a table in Excel has the focus, a custom contextual tab named Table Data is shown on the ribbon.](pnp-add-contextual-tabs-to-your-add-in.png)
```

File: Samples/office-contextual-tabs/README.md [hunk 2]
```diff
@@ -80,6 +62,12 @@ You can take the following actions to try out the add-in and the contextual tab.
 
 ## Key parts of this sample
 
+This sample inserts a table of fictitious sales data for Contoso. The data is pulled from one of two mock data sources; a mock Excel file, or a mock SQL database. The user can select which data source to use either in the task pane, or in the contextual tab.
```
Comment (ElizabethSamuel-MSFT) [inline Samples/office-contextual-tabs/README.md:65]:
```suggestion
This sample inserts a table of fictitious sales data for Contoso. The data is pulled from one of two mock data sources: a mock Excel file, or a mock SQL database. The user can select which data source to use either in the task pane, or in the contextual tab.
```

File: Samples/office-contextual-tabs/README.md [hunk 3]
```diff
@@ -181,16 +169,32 @@ If you prefer to host the web server for the sample on your computer, follow the
 
 ## Questions and comments
 
-We'd love to get your feedback about this sample. You can send your feedback to us in the Issues section of this repository. Questions about developing Office Add-ins should be posted to Stack Overflow. Ensure your questions are tagged with [office-js].
+We'd love to get your feedback about this sample. Please send your feedback to us in the Issues section of this repository. Questions about developing Office Add-ins should be posted to [Microsoft Q&A](https://docs.microsoft.com/en-us/answers/topics/office-js-dev.html) using the office-js-dev tag.
```
Comment (ElizabethSamuel-MSFT) [inline Samples/office-contextual-tabs/README.md:172]:
```suggestion
We'd love to get your feedback about this sample. Please send your feedback to us in the Issues section of this repository. Questions about developing Office Add-ins should be posted to [Microsoft Q&A](https://docs.microsoft.com/answers/topics/office-js-dev.html) using the office-js-dev tag.
```

File: Samples/office-contextual-tabs/README.md [hunk 4]
```diff
@@ -181,16 +169,32 @@ If you prefer to host the web server for the sample on your computer, follow the
...
 
-- [Office Add-ins documentation](https://docs.microsoft.com/office/dev/add-ins/overview/office-add-ins)
+- [Create custom contextual tabs in Office Add-ins](https://docs.microsoft.com/office/dev/add-ins/design/contextual-tabs)
+
+Demonstration video:
+[![YouTube video showing the contextual tab sample code and how it works](https://img.youtube.com/vi/9tLfm4boQIo/0.jpg)](https://www.youtube.com/watch?v=9tLfm4boQIo)
+
+## Solution
+
+Solution | Author(s)
```
Comment (ElizabethSamuel-MSFT) [inline Samples/office-contextual-tabs/README.md:183]: I think screen reader says "author" "s"
```suggestion
Solution | Authors
```

File: Samples/office-contextual-tabs/README.md [hunk 5]
```diff
@@ -17,44 +17,26 @@ description: "Learn how to create a contextual tab that displays on the ribbon i
...
-This sample inserts a table of fictitious sales data for Contoso. The data is pulled from one of two mock data sources; a mock Excel file, or a mock SQL database. The user can select which data source to use either in the task pane, or in the contextual tab.
-
-After the sales table is created, the sample creates a contextual tab named **Table Data**. When you select any cell or range inside the table, the contextual tab is displayed on the ribbon. When you select any cell or range outside the table, the contextual tab is hidden.
-
-The contextual tab supports commands related to working with the sales data. When you make changes you can submit them and update the mock data source. Or you can refresh the table from the mock data source.
-
 ## Run the sample
 
-You can run this sample in Excel in a browser. The add-in web files are served from this repo on GitHub.
+Run this sample in Excel in a browser. The add-in web files are served from this repo on GitHub.
```
[the remark this answers is not in the document]
Comment (davidchesnut) [reply Samples/office-contextual-tabs/README.md:40]: good catch! Should be fixed now.

File: Samples/office-contextual-tabs/README.md [hunk 6]
```diff
@@ -17,57 +17,43 @@ description: "Learn how to create a contextual tab that displays on the ribbon i
...
+- Creates a table in Excel. When the focus is inside the table, the custom tab is displayed.
+- When the focus is outside the table, the custom tab is hidden.
+
+![Screenshot that shows when a table in Excel has the focus, a custom contextual tab named Table Data is shown on the ribbon.](pnp-add-contextual-tabs-to-your-add-in.png)
 
 ## Applies to
 
--  Excel and PowerPoint on Windows 10.
+- Excel on Windows
+- Excel on web
```
Comment (ElizabethSamuel-MSFT) [inline Samples/office-contextual-tabs/README.md:31]:
```suggestion
- Excel on the web
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
One nit. Looks great otherwise.

**Pull request merged**